linkerd/docs/namer.md

```diff
@@ -147,6 +148,18 @@ The Consul namer is configured with kind `io.l5d.consul`, and these parameters:
                 (`$domain` fetched from Consul).
 
 
+For example:
+```yaml
+namers:
+- kind: io.l5d.consul
+  experimental: true
+  host: 127.0.0.1
+  port: 2181
+  includeTag: true
+  useHealthCheck: true
```

Comment:
There is already an example a few lines up in this section.  Maybe just add the `useHealthCheck` property to that example?